.github/workflows/test-latest-release.yml

```diff
@@ -0,0 +1,168 @@
+name: Test latest release
+
+on:
+  workflow_call:
+  # So we can trigger manually if needed
+  workflow_dispatch:
+  push:
+    branches:
+      - release-*
+    tags:
+      - 'v*'
+    paths-ignore:
+      - 'docs/**'
+      - '**/*.md'
+
+jobs:
+  ginkgo-test:
+    name: Tests in ginkgo
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.1.1
+      
+      - uses: actions/setup-go@v5
+        with:
+          go-version: v1.20
+          cache: true
+
+      - name: Install kubectl
+        uses: azure/setup-kubectl@v4
+        id: install
+
+      - uses: ko-build/setup-ko@v0.6
+      
+      - name: Install dependencies
+        run: |
+          curl -L https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | bash
+          wget https://github.com/kubestellar/kubeflex/releases/download/v0.4.2/kubeflex_0.4.2_linux_amd64.tar.gz
+          tar -xvf kubeflex_0.4.2_linux_amd64.tar.gz bin/kflex
+          mv bin/kflex /usr/local/bin
+          rm -fr bin kubeflex_0.4.2_linux_amd64.tar.gz
+          go install github.com/onsi/ginkgo/v2/ginkgo
+
+      - name: Run test
+        env:
+          TEST_FLAGS: ${{ github.event.inputs.testFlags }}
```

Comment:
This is broken and unused stuff copied from some other workflow, right?

Reply:
You mean the TEST_FLAGS right?  I removed them now.